src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs

```diff
@@ -2812,7 +2812,7 @@ private void GenerateHelperFunctions(TextWriter writer)
 
```

Comment:
@vexx32, your last commit had 41 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

Remoting loopback tests.Can execute command in a disconnected session
```powershell
Objects cannot be added to a closed buffer. Make sure the buffer is open for Add and Insert operations to succeed.
at <ScriptBlock>, D:\a\1\s\test\tools\Modules\HelpersRemoting\HelpersRemoting.psm1: line 225
225:     Invoke-Command @parameters

```
try/catch/finally statement parsing.Error Statement expected: <<try {1} catch [int],[char] {2} catch>>.$asts[6].Extent.Text
```powershell
Expected strings to be the same, but they were different.
Expected length: 3
Actual length:   5
Strings differ at index 0.
Expected: '{2}'
But was:  '[int]'
at <ScriptBlock>, D:\a\1\s\test\tools\Modules\HelpersLanguage\HelpersLanguage.psm1: line 125
125:             It "`$asts[$($i + 1)].Extent.Text" { $asts[$i + 1].Extent.Text | Should Be $a[$i] }
```
try/catch/finally statement parsing.Error Statement expected: <<try {1} catch [int],[char] {2} catch>>.$asts[7].Extent.Text
```powershell
Expected strings to be the same, but they were different.
Expected length: 1
Actual length:   6
Strings differ at index 0.
Expected: '2'
But was:  '[char]'
at <ScriptBlock>, D:\a\1\s\test\tools\Modules\HelpersLanguage\HelpersLanguage.psm1: line 125
125:             It "`$asts[$($i + 1)].Extent.Text" { $asts[$i + 1].Extent.Text | Should Be $a[$i] }
```
try/catch/finally statement parsing.Error Statement expected: <<try {1} catch [int],[char] {2} catch>>.$asts[8].Extent.Text
```powershell
Expected strings to be the same, but they were different.
Expected length: 1
Actual length:   3
Strings differ at index 0.
Expected: '2'
But was:  '{2}'
at <ScriptBlock>, D:\a\1\s\test\tools\Modules\HelpersLanguage\HelpersLanguage.psm1: line 125
125:             It "`$asts[$($i + 1)].Extent.Text" { $asts[$i + 1].Extent.Text | Should Be $a[$i] }
```
try/catch/finally statement parsing.Error Statement expected: <<try {1} catch [int],[char] {2} catch>>.$asts[10].Extent.Text
```powershell
Expected strings to be the same, but they were different.
Expected length: 5
Actual length:   1
Strings differ at index 0.
Expected: '[int]'
But was:  '2'
at <ScriptBlock>, D:\a\1\s\test\tools\Modules\HelpersLanguage\HelpersLanguage.psm1: line 125
125:             It "`$asts[$($i + 1)].Extent.Text" { $asts[$i + 1].Extent.Text | Should Be $a[$i] }
```